### ISSUE TITLE
Update dependencies key from "name" to "id"

### ISSUE THREAD

**Issue by benbarbour** [shemetz/ZoomPanOptions#40]
Not remotely critical, but 

```
setup.js:145 [zoom-pan-options] The module "zoom-pan-options" contains dependencies using "name" which is deprecated in favour of "id"
```

Is my last v10 console warning. :)

**Comment by shemetz** [comment]
I can do this, but then foundry v9 users will get the "k is not defined" error when trying to update :(
I wish core foundry avoided this issue by backporting the v10 package versioning code to v9, but nope.  So for now I'm going to keep this error here until Foundry V11, or at least until V10 adoption is more widespread (e.g. 90% of users).

**Comment by shemetz** [comment]
Fixed in v1.10.0
